build-priv.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Build opflex and aci-containers private images in non jenkins environment
+# 1. go get github.com/noironetworks/aci-containers
+# 2. git clone https://github.com/opendaylight/opflex
+# 3. Modify docker/Dockerfile-* and Makefile to reflect DOCKER_USER
+# 4. docker login as DOCKER_USER
+# usage: build.sh <opflex-dir> <aci-containers-dir> <docker-user>
+# example: ./build-priv.sh /home/mchalla/work/opflex-noiro /home/mchalla/go/src/github.com/noironetworks/aci-containers challa
+
+OPFLEX_DIR=$1
+ACICONTAINERS_DIR=$2
+DOCKER_USER=$3
+
```

Comment:
Instead of forcing the command line order this way, can we just use the variable directly with some defaults?

e.g. 
: ${OPFLEX_DIR:=${HOME}/work/opflex-noiro}
: ${ACICONTAINERS_DIR:=${GOPATH}/src/github.com/noironetworks/aci-containers}

It will simplify the usage.

Reply:
Yep. Changed it to use defaults if not set.